assets/js/components/UserMenu/index.js

```diff
@@ -162,6 +162,35 @@ export default function UserMenu() {
 		return null;
 	}
 
+	let accountLabel;
+
+	if ( userFullName && userEmail ) {
+		accountLabel = sprintf(
+			/* translators: Account info text. 1: User's (full) name 2: User's email address. */
+			__( 'Google Account for %1$s (Email: %2$s)', 'google-site-kit' ),
+			userFullName,
+			userEmail
+		);
+	}
+
+	if ( userFullName && ! userEmail ) {
+		accountLabel = sprintf(
+			/* translators: Account info text. 1: User's (full) name. */
+			__( 'Google Account for %1$s', 'google-site-kit' ),
+			userFullName,
+			userEmail
```

Comment:
I believe the `userEmail` variable is not needed here, right?

Reply:
Oh, you're right, thanks. Removed! 👍🏻